elasticsearch/common/src/main/java/org/apache/metamodel/elasticsearch/common/ElasticSearchUtils.java

```diff
@@ -95,17 +103,49 @@ public static QueryBuilder getExistsQuery(String fieldName) {
                 // do nothing - the ID is a client-side construct
                 continue;
             }
-            sourceProperties.add(columnName);
+            
+            final String fieldName = getValidatedFieldName(columnName);
+            final Map<String, String> propertyMap = new HashMap<>();
+            final String type = getType(column);
+            propertyMap.put("type", type);
+//            propertyMap.put("store", "true");
```

Comment:
Ah good that you mentioned it. I couldn't find the "store" attribute in newer versions of the ElasticSearch REST documentation [1]. So I think it won't work anymore ... My guess (at least my evidence so far when working with this) is that there isn't an option anymore to not store field values.

[1] https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-put-mapping.html